delta/delta_1.5.8_tbl_swmm_coefficient_friction.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE qgep_swmm.reach_coefficient_of_friction (fk_material integer, coefficient_of_friction smallint);
```

Comment:
This is the delta. Where is the creation of this table outside of the delta?

Reply:
I don't see it either, if indeed missing this should have made CI fail ! (not related to this PR, but would require a fix in the CI)

Reply:
True, the table was not created, It is created now in the file 05_data_model_extensions.sql. Indeed @olivierdalang there is a problem with the CI which has not raised this error.

Reply:
Initially we had the idea to avoid extensions in the core product, that's why we tested only base structure schemes. Since this one is officially supported by the project, it has to be tested in CI too.